security/grub-extend-pcr.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import distro, genio, process
+from avocado.utils.software_manager.manager import SoftwareManager
+
+
+class GrubExtendPCR(Test):
+    """
+    Test case for testing grub extend PCR
+    :avocado: tags=security,grub,tpm,pcr
+    """
+
+    def setUp(self):
+        '''
+        Install the basic packages
+        '''
+        # Check for basic utilities
+        val = genio.read_file("/proc/cpuinfo")
+        if 'POWER10' not in val:
+            self.cancel("Power10 LPAR is required to run this test.")
+        device_tree_path = "/proc/device-tree/vdevice/"
+        d_list = os.listdir(device_tree_path)
```

Comment:
here too if device_tree_path is not being used in more than one place then it can merge to one line
d_list = os.listdir("/proc/device-tree/vdevice/")

Reply:
Ok, resolved.